pkg/build/generator/generator.go

```diff
@@ -233,6 +233,9 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 
 	newBuild, err := g.generateBuildFromConfig(ctx, bc, request.Revision, request.Binary)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
All of the other API types should be ignored.  generateBuildFromConfig is responsible for generating the correct API error type - if you want to transform it, transform it in generateBuildFromConfig.  We should not be wrapping any error that comes out of gbfc unless it's not an API error.